tests/test_integration_fast_train.py

```diff
@@ -144,7 +144,7 @@ def test_train_timing(self):
 
         # set CacheDataset, ThreadDataLoader and DiceCE loss for MONAI fast training
         train_ds = CacheDataset(data=train_files, transform=train_transforms, cache_rate=1.0, num_workers=8)
-        val_ds = CacheDataset(data=val_files, transform=val_transforms, cache_rate=1.0, runtime_cache=True)
+        val_ds = CacheDataset(data=val_files, transform=val_transforms, cache_rate=1.0, runtime_cache=False)
```

Comment:
I double checked the test program, actually this is not a mulit-gpu distributed training, so no need to change it.
Could you please help restore it?
Sorry for my mistake.

Thanks in advance.

Reply:
I've updated it, thank you

Reply:
I see you didn't restore the test in the latest commit.
Could you please help set `runtime_cache=True` as the dev branch for this line?

Thanks.

Reply:
I removed **runtime_cache** from **test_integration_fast_train.py**, because here we use **num_workers=0** (with ThreadDataLoader).  So I assume for this case we do not want to use runtime_cache, right ? @Nic-Ma

Reply:
Hi @myron ,

I think we only dropped the support for "DDP + GPU cache + runtime cache", right?
This test is not DDP, so `runtime_cache=True` should work here, and it's a good test case.
What do you think?

Thanks.

Reply:
Oh, okay, I'll add it back

Reply:
done